core/src/main/java/com/scalar/db/storage/jdbc/JdbcTableMetadataManager.java

```diff
@@ -31,11 +33,23 @@
 public class JdbcTableMetadataManager implements TableMetadataManager {
```

Comment:
According to the spotbugs report, we need to suppress or fix the warning:
https://4146-151700458-gh.circle-artifacts.com/0/gradle_spotbugs_reports_for_core/spotbugs/main.html

Reply:
I think this one is a false positive to as even though is it dynamically generated. No external input is used to build the query.